### ISSUE TITLE
Property attributes missing in export

### ISSUE THREAD

**Issue by ivanovych666** [orientechnologies/orientdb#6148]
## Expected behavior and actual behavior

Expected behavior is to have all property attributes in exported schema.

Actual behavior: in exported schema property attributes DEFAULT and REGEXP are missing (DEFAULT is present in 2.2.0)

## Steps to reproduce the problem

create or connect to database

```
CREATE CLASS ClassName EXTENDS V
CREATE PROPERTY ClassName.propertyREGEXP STRING
ALTER PROPERTY ClassName.propertyREGEXP REGEXP [a-z]
CREATE PROPERTY ClassName.propertyDEFAULT STRING
ALTER PROPERTY ClassName.propertyDEFAULT DEFAULT "defaultValue"
```

then create export file

`EXPORT DATABASE /path/to/file`

or 

`EXPORT SCHEMA /path/to/file`

### Runninng Mode
- [ ] Embedded, using PLOCAL access mode
- [ ] Embedded, using MEMORY access mode
- [x] Remote

### Misc
- [ ] I have a distributed setup with multiple servers. How many? 
- [x] I'm using the Enterprise Edition

### OrientDB Version
- [ ] v2.0.x - Please specify last number: 
- [x] v2.1.x - Please specify last number: 2.1.16
- [x] v2.2.x - Please specify last number: 2.2.0

### Operating System
- [x] Linux
- [ ] MacOSX
- [ ] Windows
- [ ] Other Unix
- [ ] Other, name? 

### Java Version
- [ ] 6
- [ ] 7
- [x] 8

**Comment by lvca** [comment]
This fix is in conflict with other parts. I'm sorry I have to postpone it.

**Comment by luigidellaquila** [comment]
Fixed in develop branch (2.1.1)

Thanks

Luigi
